src/util/cfg/cfg.ts

```diff
@@ -524,24 +524,25 @@ function cfgUnaryOp(unary: RNodeWithParent, operand: ControlFlowInformation): Co
 
 
 function cfgExprList(_node: RNodeWithParent, expressions: ControlFlowInformation[]): ControlFlowInformation {
```

Comment:
not too sure whether we just just switch to basic blocks

Reply:
Tracked in #812